docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -48,28 +41,30 @@ The component can be uncontrolled or controlled
 
 ## Responsiveness
 
-The Date Picker component is designed and optimized for the device it runs on.
+The component is available in four variants:
 
-- The `MobileNextDatePicker` component works best for touch devices and small screens.
-- The `DesktopNextDatePicker` component works best for mouse devices and large screens.
+- The `DesktopNextDatePicker` component which works best for mouse devices and large screens.
+  It renders the views inside a popover and allows editing values directly inside the field.
 
-By default, the `NextDatePicker` component renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
-This can be customized with the `desktopModeMediaQuery` prop.
+- The `MobileNextDatePicker` component which works best for touch devices and small screens.
+  It renders the view inside a modal and does not allow editing values directly inside the field.
 
-There are certain caveats when testing pickers, please refer to [this section](/x/react-date-pickers/getting-started/#testing-caveats) for more information.
+- The `NextDatePicker` component which renders `DesktopNextDatePicker` or `MobileNextDatePicker` depending on the device it runs on.
 
-{{"demo": "ResponsiveDatePickers.js"}}
+  By default, it renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
+  This can be customized with the `desktopModeMediaQuery` prop.
 
-## Static mode
+- The `StaticDatePicker` component which renders without the popover/modal and field.
```

Comment:
This modification is not applied to Time and Date Time picker pages

Reply:
As mentionned [here](https://github.com/mui/mui-x/pull/7339#discussion_r1059877523), I'd like to confirm the structure before applying it to all the pages

Reply:
It is now applied everywhere